.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,9 +8,14 @@
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Test case or documentation update (no impact on source code)
+- [ ] Test automation update Unit or/and Functional (no changes in operator code)
+- [ ] Product update
+
+<!-- Different check list per type of change? -->
 
 ## Checklist
 - [ ] This change requires a documentation update <!-- Update JIRA with Affects -> Documentation -->
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have added a test case that will be used to verify my changes 
-- [ ] Verified independently on a cluster by reviewer
\ No newline at end of file
+- [ ] Verified independently on a cluster by reviewer
```

Comment:
I wonder if we should also have options `Verified locally by creator of PR`?
-

Reply:
I agree, On a side note, we'd need to start enforcing the use of these tickboxes by reviewers, unfortunately I haven't seen many people using it :(